go/database/mpt/forest_test.go

```diff
@@ -256,6 +256,7 @@ func TestForest_CreatingAccountInfo_Fails(t *testing.T) {
 					if err != nil {
 						t.Fatalf("failed to open forest: %v", err)
 					}
+					defer forest.Close()
```

Comment:
here and in other cases I would like to see assertions for the result of `Close() `as well